src/main/java/io/github/robwin/swagger/test/ConsumerDrivenValidator.java

```diff
@@ -96,8 +96,6 @@ private void validateDefinitions(Map<String, Model> actualDefinitions, Map<Strin
                     validateDefinition(definitionName, actualDefinition, expectedDefinition);
                 }
             }
-        }else{
-            softAssertions.assertThat(actualDefinitions).as("Checking Definitions").isNullOrEmpty();
```

Comment:
these lines should have never been added to this file, but they were here, because the code in this file seems to be copied from DocumentationDrivenValidator (where these lines still make sense).
Reason: when checking design api-doc against implementation using `.satisfiesContract(designApiDoc)`, 
(not with `.isEqualTo(designApiDoc)`), 
then we shouldn't care how many definitions the implementation api-doc has when designApiDoc has no definitions.